Dockerfile.mri.erb

```diff
@@ -135,8 +135,7 @@ RUN bash -c " \
 axrubies = if platform =~ /x64-mingw-ucrt/
   [
     # Rubyinstaller-3.1.0+ is platform x64-mingw-ucrt
-    # parallel=false because 3.2.0-rc1 has undefined references when using a high -j arg
-    ["3.2.0-rc1:3.1.0", "3.1.3", false],
+    ["3.2.0:3.1.0", "3.1.3", true],
```

Comment:
The parallel cross build of x64-mingw-ucrt has been fixed upstream?

Reply:
I couldn't get it to break on my local machine, whereas it broke every time locally with 3.2.0-rc1. I don't know what, if anything, changed between the rc and the final release, only that it was working for me.

Reply:
Note that this was set to false only recently in 611a648